app/pages/home-for-user/my-builds.jsx

```diff
@@ -101,8 +88,7 @@ const MyBuildsSection = React.createClass({
 
         <div className="project-card-list">
           {this.state.projects.map((project) => {
-            const avatarSrc = !!this.state.avatars[project.id] ? this.state.avatars[project.id].src : null;
-            return <ProjectCard key={project.id} project={project} imageSrc={avatarSrc} href={`/lab/${project.id}`} />;
+            return <ProjectCard key={project.id} project={project} href={`/lab/${project.id}`} />;
```

Comment:
@srallen passing the project prop in here has a cut down resource response, e.g. no project.tags attributes. so when following the link the lab page falls over as it crashes trying to split the tags